tensorflow/lite/kernels/internal/common.h

```diff
@@ -263,6 +263,30 @@ inline void gen_lut(const std::function<double(double)>& func, double min,
       std::min(std::max(TfLiteRound(func(max) * 32768.0), -32768.0), 32767.0);
 }
 
+// generate INT16 LUT for function(), e.g., table exp(x) and 1/(1+x) used in
+// softmax
+inline void gen_lut(const std::function<float(float)>& func, float min,
+                    float max, int16_t* table, const int num) {
```

Comment:
the std::function param is resulting in dynamic memory allocation. I think changing it to be a function pointer might do the trick.

Also, please document the parameters of the function.

Reply:
Yes, changing to function pointer seems to be enough.